### PR TITLE
Small restructure, README improvement, add boreas cleaner

### PR DESCRIPTION
**Pull request by brandonsturgeon**

### CODE REVIEW

File: lua/cfc_gmod_scripts/net_remover/sv_remover.lua [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
     "pac3_test_suite_backdoor",
     "pac3_test_sutie_backdoor"
 }
 
-hook.Add( "InitPostEntity", "GMS_RemoveBadNetReceivers", function()
+local function enable()
     for _, name in ipairs( netToRemove ) do
-        if net.Receivers[name] then
-            print( "[CFC Gmod Scripts] Removing net receiver for " .. name )
+        if receiver then
```
Comment (wrefgtzweve) [inline lua/cfc_gmod_scripts/net_remover/sv_remover.lua:12]: receiver is never defined

File: lua/cfc_gmod_scripts/net_remover/sv_remover.lua [hunk 2]
```diff
@@ -1,14 +1,19 @@
+net.GMScripts_RemovedReceivers = net.GMScripts_RemovedReceivers or {}
+
+local originals = net.GMScripts_RemovedReceivers
 local netToRemove = {
```
[the remark this answers is not in the document]
Comment (brandonsturgeon) [reply lua/cfc_gmod_scripts/net_remover/sv_remover.lua:4]: separate script maybe?

[the remark this answers is not in the document]
Comment (wrefgtzweve) [reply lua/cfc_gmod_scripts/net_remover/sv_remover.lua:4]: not a real use for us, might not even be worth adding

File: lua/cfc_gmod_scripts/net_remover/sv_remover.lua [hunk 3]
```diff
@@ -1,14 +1,21 @@
...
+
+        if receiver then
+            originals[name] = originals[name] or receiver
             net.Receivers[name] = nil
         end
     end
-end )
+end
+
+hook.Add( "InitPostEntity", "GM_Scripts_NetRemover", enable )
```
Comment (wrefgtzweve) [inline lua/cfc_gmod_scripts/net_remover/sv_remover.lua:21]: hook name is `CFC_GmodScripts_PlayerPickupFix` in another script, they should follow the same style across the addon.

**Pull request closed without merging**